examples/evaluation/How_to_eval_abstractive_summarization.ipynb

```diff
@@ -0,0 +1,608 @@
+     "x": 0,
+     "y": 0
+    },
+    "deepnote_cell_type": "code",
+    "deepnote_to_be_reexecuted": false,
+    "execution_millis": 7700,
+    "execution_start": 1692083249280,
+    "source_hash": "ab0afee3"
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<style  type=\"text/css\" >\n",
+       "#T_58281_row0_col0,#T_58281_row1_col0,#T_58281_row1_col1,#T_58281_row2_col0,#T_58281_row3_col0{\n",
+       "            background-color:  lightgreen;\n",
+       "        }#T_58281_row0_col1,#T_58281_row2_col1,#T_58281_row3_col1{\n",
+       "            background-color:  white;\n",
+       "        }</style><table id=\"T_58281_\" ><thead>    <tr>        <th class=\"index_name level0\" >Summary Type</th>        <th class=\"col_heading level0 col0\" >Summary 1</th>        <th class=\"col_heading level0 col1\" >Summary 2</th>    </tr>    <tr>        <th class=\"index_name level0\" >Evaluation Type</th>        <th class=\"blank\" ></th>        <th class=\"blank\" ></th>    </tr></thead><tbody>\n",
+       "                <tr>\n",
+       "                        <th id=\"T_58281_level0_row0\" class=\"row_heading level0 row0\" >Coherence</th>\n",
+       "                        <td id=\"T_58281_row0_col0\" class=\"data row0 col0\" >5</td>\n",
+       "                        <td id=\"T_58281_row0_col1\" class=\"data row0 col1\" >3</td>\n",
+       "            </tr>\n",
+       "            <tr>\n",
+       "                        <th id=\"T_58281_level0_row1\" class=\"row_heading level0 row1\" >Consistency</th>\n",
+       "                        <td id=\"T_58281_row1_col0\" class=\"data row1 col0\" >5</td>\n",
+       "                        <td id=\"T_58281_row1_col1\" class=\"data row1 col1\" >5</td>\n",
+       "            </tr>\n",
+       "            <tr>\n",
+       "                        <th id=\"T_58281_level0_row2\" class=\"row_heading level0 row2\" >Fluency</th>\n",
+       "                        <td id=\"T_58281_row2_col0\" class=\"data row2 col0\" >3</td>\n",
+       "                        <td id=\"T_58281_row2_col1\" class=\"data row2 col1\" >2</td>\n",
+       "            </tr>\n",
+       "            <tr>\n",
+       "                        <th id=\"T_58281_level0_row3\" class=\"row_heading level0 row3\" >Relevance</th>\n",
+       "                        <td id=\"T_58281_row3_col0\" class=\"data row3 col0\" >5</td>\n",
+       "                        <td id=\"T_58281_row3_col1\" class=\"data row3 col1\" >4</td>\n",
+       "            </tr>\n",
+       "    </tbody></table>"
+      ],
+      "text/plain": [
+       "<pandas.io.formats.style.Styler at 0x7f1933992370>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Evaluation prompt template based on G-Eval\n",
+    "EVALUATION_PROMPT_TEMPLATE = \"\"\"\n",
+    "You will be given one summary written for an article. Your task is to rate the summary on one metric.\n",
```

Comment:
Before diving into this code, it could be nice to give an overview of what G-Eval does, like which are all the metrics and how they're combined.

Reply:
agreed - added a blurb to break it down into the 3 components.